src/js/Pickers/CalendarHeader.js

```diff
@@ -129,14 +129,18 @@ export default class CalendarHeader extends PureComponent {
             disabled={isPreviousDisabled}
             className="md-calendar-control"
             iconEl={previousIcon}
+            aria-label="Previous Month"
```

Comment:
Do you think these labels could be props that default to "Previous Month" and "Next Month"? Maybe `previousLabel` and `nextLabel`? It'll help being able to change the label based on the current language.

Reply:
They are always months, yes? I like the explicit-ness of `previousMonthLabel` and `nextMonthLabel` if that's alright by you.

Reply:
Yep, sounds good to me!